.husky/pre-commit

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
```

Comment:
`hint: The '.husky/commit-msg' hook was ignored because it's not set as executable.`

Currently seeing this. Ideally we'd like this to work right out of the box, but if that's not possible, you should document the steps on how to set it as an executable

Reply:
when do you see this after doing the commit or push?

Reply:
when I commit! also when I say document steps you can probably just add a comment or two to the husky files